### PR TITLE
Recipes and time display

### PR DESCRIPTION
**Pull request by Tshireletso46**
Rendered recipe list in the home page.
Added time display component.

### CODE REVIEW

File: components/recipe-collection/RecipeList.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            setRemainingRecipes(initialRemainingRecipes);
+          }catch (error) {
+        console.error('Error fetching recipes:', error);
+      }
+    };
+
+    callViewRecipes();
+  }, []);
+
+  console.log(recipes)
```
Comment (NelsonMALGAS) [inline components/recipe-collection/RecipeList.js:28]: Please remove the console log

Comment (Tshireletso46) [reply components/recipe-collection/RecipeList.js]: Okay

**Review by kgakatsikeamogetswe — CHANGES_REQUESTED** [review]
Everything looks fine, just the console.log on the RecipeList.js, lastly I recommend you use this "../" instead of "@/"

**Review by NelsonMALGAS — APPROVED** [review]

**Review by kgakatsikeamogetswe — APPROVED** [review]
Everything looks fine

**Review by NelsonMALGAS — APPROVED** [review]
All looks good

**Pull request merged**